build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# check if virtualenv already exists
+if [ -d "$HOME/ubportsdocsenv" ]; then
+  echo "Build environment found"
+  . ~/ubportsdocsenv/bin/activate
+else
+  echo "No build environment found. Creating..."
+  # install prerequisites
+  sudo apt install python-pip
+  sudo pip install virtualenv
+  # create virtualenv
+  virtualenv ~/ubportsdocsenv
+  . ~/ubportsdocsenv/bin/activate
+  # install build tools
+  sudo pip install sphinx sphinx_rtd_theme recommonmark
```

Comment:
Luckily we don't need recommonmark any more. Should be changed in the Contribute doc, shame on me.